src/extension/wps/src/main/java/au/org/emii/aggregator/variable/NetcdfVariableAdapter.java

```diff
@@ -0,0 +1,61 @@
+package au.org.emii.aggregator.variable;
+
+import ucar.ma2.Array;
+import ucar.ma2.DataType;
+import ucar.ma2.InvalidRangeException;
+import ucar.nc2.Attribute;
+import ucar.nc2.Dimension;
+import ucar.nc2.constants.AxisType;
+import ucar.nc2.dataset.CoordinateAxis;
+import ucar.nc2.dataset.VariableDS;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Convert a ucar.m2.VariableDS to the NetcdfVariable interface used in this library
+ */
+public class NetcdfVariableAdapter extends AbstractVariable {
```

Comment:
Need to think about this some more.  If the adapter is built into either the unpacker or subsetter then they will be forced to deal with concrete Unidata Variables making unit testing more difficult and requiring the use of a particular class.  I was trying to keep this generic and useful for other use cases.  Will handle any changes seperately to this PR.